test/test_transforms_tensor.py

```diff
@@ -858,3 +858,33 @@ def test_gaussian_blur(device, channels, meth_kwargs):
         agg_method="max",
         tol=tol,
     )
+
+
+@pytest.mark.parametrize("device", cpu_and_gpu())
+@pytest.mark.parametrize(
+    "meth_kwargs",
```

Comment:
NIT: just parametrize over `fill` and use `meth_kwargs = {"fill": fill}` like in other tests

Reply:
Fair point.